exercises/forth/canonical-data.json

```diff
@@ -0,0 +1,219 @@
+{
+  "#": [
+    "The cases are split into multiple sections, all with the same structure.",
+    "In all cases, the `expected` key is the string representation of the resulting stack",
+    "if all numbers on it are printed (top-most value on the right), separated by a space."
```

Comment:
Time to question assumptions. why is it done this way? It was done this way in https://github.com/exercism/exercism.io/pull/1188 (the first time this exercise appeared), and I guess everyone else did the same. But why not check an array/list of values on the stack? Instead of `"expected": "1 2 3 4 5"`, why not `"expected": [1, 2, 3, 4, 5]` ?